priv/repo/migrations/20160913180125_set_default_state_for_surveys.exs

```diff
@@ -4,13 +4,13 @@ defmodule Ask.Repo.Migrations.SetDefaultStateForSurveys do
 
   def up do
     alter table(:surveys) do
-      modify :state, :string, default: "pending"
+      modify :state, :string, default: :pending
```

Comment:
Do we need to change the old migration files? Do the changes in the model force us to do it? If that's the case, OK, maybe it's our only option. In any case, I feel odd doing it. I'd always try first to add new migration files with the required changes.

Reply:
If we want to run the migrations using current code, yes, we probably do.

For this specific case of a default SQL value, we probably could be using a string.

Reply:
But just below we should be using the atom.